instrumentation/spring/spring-boot-autoconfigure/src/test/java/io/opentelemetry/instrumentation/spring/autoconfigure/internal/instrumentation/scheduling/SchedulingInstrumentationAspectTest.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.internal.instrumentation.scheduling;
+
+import static io.opentelemetry.api.trace.SpanKind.INTERNAL;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.equalTo;
+import static io.opentelemetry.sdk.testing.assertj.TracesAssert.assertThat;
+import static io.opentelemetry.semconv.incubating.CodeIncubatingAttributes.CODE_FUNCTION;
+import static io.opentelemetry.semconv.incubating.CodeIncubatingAttributes.CODE_NAMESPACE;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.instrumentation.testing.junit.LibraryInstrumentationExtension;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.internal.DefaultConfigProperties;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.data.StatusData;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.aop.aspectj.annotation.AspectJProxyFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.scheduling.annotation.Schedules;
+
+class SchedulingInstrumentationAspectTest {
+
+  @RegisterExtension
+  static final LibraryInstrumentationExtension testing = LibraryInstrumentationExtension.create();
+
+  private InstrumentationSchedulingTester schedulingTester;
+  private String unproxiedTesterSimpleClassName;
+  private String unproxiedTesterClassName;
+
+  SpringSchedulingInstrumentationAspect newAspect(
+      OpenTelemetry openTelemetry, ConfigProperties configProperties) {
+    return new SpringSchedulingInstrumentationAspect(openTelemetry, configProperties);
+  }
+
+  @BeforeEach
+  void setup() {
+    InstrumentationSchedulingTester unproxiedTester =
+        new InstrumentationSchedulingTester(testing.getOpenTelemetry());
+    unproxiedTesterSimpleClassName = unproxiedTester.getClass().getSimpleName();
+    unproxiedTesterClassName = unproxiedTester.getClass().getName();
+
+    AspectJProxyFactory factory = new AspectJProxyFactory();
```

Comment:
Are you trying to enable the AspectJ support with Spring? In this case, you could use `@EnableAspectJAutoProxy` (https://docs.spring.io/spring-framework/reference/core/aop/ataspectj/aspectj-support.html)